features/checkout/paying_for_order/accessing_order_right_after_completing_checkout.feature

```diff
@@ -1,21 +1,20 @@
 @paying_for_order
 Feature: Having good number of items in changing payment method page
     In order to verify that I am changing the payment method of correct order
-    As a Customer
+    As a Guest
     I want to see correct details about my order on changing the payment method page
 
     Background:
         Given the store operates on a single channel in "United States"
-        And there is a user "john@example.com" identified by "password123"
         And the store allows paying "Offline"
         And the store allows paying "Cash on delivery"
         And the store has a product "PHP T-Shirt" priced at "$19.99"
         And the store ships everywhere for free
-        And I am logged in as "john@example.com"
 
     @ui
     Scenario: Seeing correct quantity on payment retry page
         Given I have added 2 products "PHP T-Shirt" to the cart
+        And I complete addressing step with email "john@example.com" and "United States" based shipping address
         And I have proceeded selecting "Cash on delivery" payment method
         And I have confirmed order
         When I go to order details
```

Comment:
How can we go to order details, if we are not a registered customer?

Reply:
I need pass address steep